kernel/conversion.ml

```diff
@@ -431,10 +437,10 @@ and eqappr cv_pb l2r infos (lft1,st1) (lft2,st2) cuniv =
     | (FFlex fl1, FFlex fl2) ->
       (try
          let nargs = same_args_size v1 v2 in
-         let cuniv = conv_table_key infos.cnv_inf ~nargs fl1 fl2 cuniv in
+         let cuniv = conv_table_key infos ~nargs fl1 fl2 cuniv in
          let () = if irr_flex infos.cnv_inf fl1 then raise NotConvertible (* trigger the fallback *) in
          convert_stacks l2r infos lft1 lft2 v1 v2 cuniv
-       with NotConvertible | UGraph.UniverseInconsistency _ ->
+       with NotConvertible | NotConvertibleTrace _ ->
```

Comment:
It doesn't need to be in this PR, but in the case where neither fflex is unfoldable we may want to reraise this error instead of raising NotConvertible and dropping any further information.